src/commands/moderation/prefixCommands/functions/addChannelPermission.ts

```diff
@@ -0,0 +1,112 @@
+import { ChatInputCommandInteraction, Colors, TextChannel, User } from 'discord.js';
+import { constantsConfig, getConn, PrefixCommand, Logger, makeEmbed, refreshSinglePrefixCommandCache } from '../../../../lib';
+
+const noConnEmbed = makeEmbed({
+    title: 'Prefix Commands - Add Channel - No Connection',
+    description: 'Could not connect to the database. Unable to add the prefix command channel.',
+    color: Colors.Red,
+});
+
+const noCommandEmbed = (command: string) => makeEmbed({
+    title: 'Prefix Commands - Add Channel - No Command',
+    description: `Failed to add the prefix command channel for command ${command} as the command does not exist or there are more than one matching.`,
+    color: Colors.Red,
+});
+
+const failedEmbed = (command: string, channel: string) => makeEmbed({
+    title: 'Prefix Commands - Add Channel Permission - Failed',
```

Comment:
```suggestion
    title: 'Prefix Commands - Add Channel - Failed',
```

The rest just say 'Add Channel'